Sources/MessagingPush/MessagingPush+NotificationCenter.swift

```diff
@@ -42,6 +59,24 @@ public extension MessagingPush {
         return false
     }
 
+    func trackMetric(
+        notificationContent: UNNotificationContent,
+        event: Metric,
+        onComplete: @escaping (Result<Void, CustomerIOError>) -> Void
+    ) {
+        guard let deliveryID: String = notificationContent.userInfo["CIO-Delivery-ID"] as? String else {
+            onComplete(Result.success(()))
+            return
+        }
+
+        guard let deviceToken: String = notificationContent.userInfo["CIO-Delivery-Token"] as? String else {
+            onComplete(Result.success(()))
+            return
+        }
+
+        trackMetric(deliveryID: deliveryID, event: event, deviceToken: deviceToken, onComplete: onComplete)
```

Comment:
I find `deviceToken` named parameter confusing because it is the same naming we give FCM and APN device tokens. I am wondering if customers will find it confusing? 

The [api body](https://customer.io/docs/api/#operation/pushMetrics) names it `deliver_id` while the push JSON payload's name is `CIO-Delivery-Token` as we see here. 

Any suggestions?

Reply:
`CIO-Delivery-ID` is a delivery ID, and `CIO-Delivery-Token` _is_ the deviceToken of the app instance that received the push. I assume we did it this way in case the customer wasn't storing a token locally in their integratino. IMO using the one on the headers is probs the right move, but it wouldn't matter either way.

Reply:
in the api we call it `delivery_id` and `device_id` so we could I guess use `deviceID` but since it is actually a deviceToken, felt like it would be weird to introduce a new term into the SDK